ruby-challenges.rb

```diff
@@ -2,27 +2,58 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
-
 reposts1 = 7
 # Expected output: '7 is odd'
 reposts2 = 42
 # Expected output: '42 is even'
 reposts3 = 221
 # Expected output: '221 is odd'
 
+# Psuedo Code
+# Create a function called is_even_odd
+# Function will take a number as an input
+# Use the .even? method to return a boolean "true/false"
+# Use a conditional to that compares the boolean value returned from the .even? method
+# If number is true then return "#{num} is even"
+# if number is false then return "#{num} is odd"
+
+def is_even_odd num
+    if num.even? == true
+        "#{num} is even"
+    else 
+        "#{num} is odd"
+    end
+end
 
```

Comment:
Good use of the `.even` method! You don't actually need the `== true` portion there at all, because the method is returning true or false by default.